### PR TITLE
Endpoints and parameters added to support applications.

### PR DESCRIPTION
**Pull request by winder**

### CODE REVIEW

File: utils/create_indexer_integration_goldens.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```
Comment (EvanJRichard) [inline utils/create_indexer_integration_goldens.sh:1]: Cool, I like this :+1:

File: features/integration/indexer.feature [hunk 2]
```diff
@@ -407,6 +438,53 @@ Feature: Indexer Dataset 1
...
+      | 2       | 22             |  v2indexerclient_responsejsons/indexer_v2_app_lookup_22.json      |
+      | 2       | 70             |  v2indexerclient_responsejsons/indexer_v2_app_lookup_70.json      |
+
+  #
+  # /transactions
+  #  When I use <indexer> to search for transactions with <limit>, "<note-prefix>", "<tx-type>", "<sig-type>", "<tx-id>", <round>, <min-round>, <max-round>, <asset-id>, "<before-time>", "<after-time>", <currency-gt>, <currency-lt>, "<address>", "<address-role>", "<exclude-close-to>" and token "<token>"
+  #
+  @indexer.applications
+  Scenario Outline: /transactions?everything
+    #When I use <indexer> to search for transactions with <limit>, "<note-prefix>", "<tx-type>", "<sig-type>", "<tx-id>", <round>, <min-round>, <max-round>, <asset-id>, "<before-time>", "<after-time>", <currency-gt>, <currency-lt>, "<address>", "<address-role>", "<exclude-close-to>", <application-id> and token "<token>"
```
Comment (EvanJRichard) [inline features/integration/indexer.feature:470]: Is this comment to show what fields are filled in the non-comment line below?

File: README.md [hunk 3]
```diff
@@ -11,30 +11,29 @@ We have different feature files for unit and integration tests. The unit tests s
...
-| @unit.applications     | Application endpoints added to Algod and Indexer. |
-| @unit.dryrun           | Dryrun endpoint added to Algod. |
-| @unit.indexer          | Indexer REST API unit tests. |
-| @unit.indexer.rekey    | Rekey endpoints added to Algod and Indexer |
-| @unit.offline          | The first unit tests we wrote for cucumber. |
-| @unit.transactions     | Transaction golden tests. |
-| @unit.rekey            | Rekey Transaction golden tests. |
-| @unit.responses        | REST Client Response serialization tests. |
-| @unit.tealsign         | Test TEAL signature utilities. |
+| tag                   | description |
```
Comment (EvanJRichard) [inline README.md:14]: Any reason for removing the header bar? I could take it or leave it, just curious.

Comment (winder) [reply README.md:14]: oops, bad merge!

**Pull request merged**